docs/docs/identity-providers/google.md

```diff
@@ -122,7 +122,21 @@ Next, we need to give that service account permissionson the GSuite / Workspace
 
 ![Google create service account](./img/google/google-gsuite-add-scopes.png)
 
-Your [environmental variables] should look something like this.
+
+Your `config.yaml` values or [environment variables] should look something like this:
+
+::::: tabs
+:::: tab config.yaml
+
+```yaml
+idp-provider: "google"
+idp-client-id: "yyyy.apps.googleusercontent.com"
+idp-client-secret: "xxxxxx"
+idp-service-account: "zzzz" # output of `base64 -i service-account-key.json`
```

Comment:
IMO, this should explicitly mention the need to add `impersonate_user` to the JSON, since the suggested command won't work without it.  https://www.pomerium.com/docs/identity-providers/google.html#set-the-impersonating-user